lib/key.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const ObjectHash = require('object-hash');
+
+exports.generate = (Model, request, uniqueKeyFn) => {
+  let tableName = '';
+  let filterHash = '';
+  let uniqueKey = '';
+
+  if (Model && Model.prototype) {
+    tableName = Model.prototype.tableName;
+  }
```

Comment:
Could we add validation in `index.js` that bails out if the model provided is not a bookshelf model? It seems like it would be more convenient to just validate it at the source instead of having to check so many times later on.